kits/multisig/test/multisig.js

```diff
@@ -77,264 +77,285 @@ contract('Multisig Kit', accounts => {
-                const script2 = encodeCallScript([action2])
-                const r = await tokenManager.forward(script2, { from: signer1 })
-                voteId = getVoteId(r)
-            })
-
-            it('has correct state', async() => {
-                const [isOpen, isExecuted, startDate, snapshotBlock, requiredSupport, minQuorum, y, n, totalVoters, execScript] = await voting.getVote(voteId)
-
-                assert.isTrue(isOpen, 'vote should be open')
-                assert.isFalse(isExecuted, 'vote should be executed')
-                assert.equal(snapshotBlock.toString(), await getBlockNumber() - 1, 'snapshot block should be correct')
-                assert.equal(requiredSupport.toString(), multisigSupport.toString(), 'min quorum should be app min quorum')
-                assert.equal(minQuorum.toString(), multisigSupport.toString(), 'min quorum should be app min quorum')
-                assert.equal(y, 0, 'initial yea should be 0')
-                assert.equal(n, 0, 'initial nay should be 0')
-                assert.equal(totalVoters.toString(), signers.length, 'total voters should be number of signers')
-                assert.equal(execScript, script, 'script should be correct')
-            })
-
-            it('holder can vote', async () => {
-                await voting.vote(voteId, false, true, { from: signer2 })
-                const state = await voting.getVote(voteId)
-
-                assert.equal(state[7].toString(), 1, 'nay vote should have been counted')
-            })
-
-            it('holder can modify vote', async () => {
-                await voting.vote(voteId, true, true, { from: signer2 })
-                await voting.vote(voteId, false, true, { from: signer2 })
-                await voting.vote(voteId, true, true, { from: signer2 })
-                const state = await voting.getVote(voteId)
-
-                assert.equal(state[6].toString(), 1, 'yea vote should have been counted')
-                assert.equal(state[7], 0, 'nay vote should have been removed')
-            })
-
-            it('throws when non-holder tries to create a vote directly', async () => {
-                const action = { to: executionTarget.address, calldata: executionTarget.contract.execute.getData() }
-                script = encodeCallScript([action])
-                try {
-                    await voting.newVote(script, 'metadata', { from: nonHolder })
-                } catch (err) {
-                    assert.equal(err.receipt.status, 0, "It should have thrown")
-                    return
+    // Test when organization is created in one call with `newTokenAndInstance()` and in
+    // two calls with `newToken()` and `newInstance()`
+    const creationStyles = ['single', 'separate']
+    for (const creationStyle of creationStyles) {
+        context(`Creation through ${creationStyle} transaction`, () => {
```

Comment:
@bingen I'm trying to add a parameterized test so the same tests we had before can be run for organizations created through `newTokenAndInstance()`.